pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -125,6 +126,27 @@ let
   luaPackages = neovim-unwrapped.lua.pkgs;
 in
 {
+  corePlugins = symlinkJoin {
+    name = "core-vim-plugins";
+    paths = with self; [
+      lazy-nvim
+      nvim-cmp
+      nvim-lspconfig
+      nvim-treesitter
+      plenary-nvim
+      vim-airline
+      vim-fugitive
+      vim-surround
```

Comment:
I think I would add all the plugins manager because breaking those could be a huge pain for users:
paq / plug / mini.something (never used any mini plugin) / the new packer ? (https://github.com/lewis6991/pckr.nvim)

Reply:
Agree, makes sense. 

- packer-nvim
- mini-deps
- pckr (not packaged yet?)
- lazy.nvim